man/kernel-install.xml

```diff
@@ -61,15 +62,15 @@
     <para>The following commands are understood:</para>
     <variablelist>
       <varlistentry>
-        <term><command>add <replaceable>KERNEL-VERSION</replaceable> <replaceable>KERNEL-IMAGE</replaceable></command></term>
+        <term><command>add <replaceable>KERNEL-VERSION</replaceable> <replaceable>KERNEL-IMAGE</replaceable> <replaceable>INITRD-FILE</replaceable></command></term>
```

Comment:
Maybe, `[]` should be added around `INITRD-FILE`.

Reply:
Hiya, these have been added, but as mentioned below, it's now inconsistent (since `KERNEL-IMAGE` is marked as optional further up, but doesn't have the `[]`).  Given I just use the third argument, I assume the appropriate style for the command would be `[KERNEL-IMAGE [INITRD-FILE]]`?